draft_python/build.py

```diff
@@ -0,0 +1,30 @@
+import sys
+from mtg_draft_ai import draftlog, deckbuild, display, synergy
+from mtg_draft_ai.api import read_cube_toml
+
+log_file = sys.argv[1]
+output_file = 'build.html' if len(sys.argv) < 3 else sys.argv[2]
```

Comment:
At this point should we have these generated files maybe all grouped under some folder by default, like 'output/' or something? Just so they don't pollute the primary directory with a bunch of files.

This goes for other files generated from other python modules too.

Reply:
Sounds good. I added a "drafts/" folder as the output for trials.py, but "output/" sounds better in general.